### PR TITLE
services/tar: Migrate to Factory init.

### PR DESCRIPTION
**Pull request by Joey2132**
Migrate to Factory init.

### CODE REVIEW

File: services/tar/internal/cmd/meta.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	Storage: def.Storage{
+		Features: types.StorageFeatures{
+			WriteEmptyObject: true,
+
+			Delete:   true,
+			List:     true,
+			Metadata: true,
+			Read:     true,
+			Stat:     true,
+			Write:    true,
```
Comment (JinnyYi) [inline services/tar/internal/cmd/meta.go:28]: In fact, only `List`, `Read` and `Stat` are currently implemented.

Comment (Joey2132) [reply services/tar/internal/cmd/meta.go:28]: Got it, I will check it.

File: services/tar/internal/cmd/meta.go [hunk 2]
```diff
@@ -20,12 +20,9 @@ var Metadata = def.Metadata{
 		Features: types.StorageFeatures{
 			WriteEmptyObject: true,
```
Comment (JinnyYi) [inline services/tar/internal/cmd/meta.go:21]: `WriteEmptyObject` should also be `false`.

File: services/tar/utils.go [hunk 3]
```diff
@@ -12,10 +12,44 @@ import (
...
+
+	defaultPairs types.DefaultServicePairs
+	features     types.ServiceFeatures
+
+	types.UnimplementedServicer
+}
+
+// String implements Servicer.String
+func (s *Service) String() string {
+	return fmt.Sprintf("Servicer cephfs")
```
Comment (JinnyYi) [inline services/tar/utils.go:28]: Service name should be `tar`.

**Review by JinnyYi — APPROVED** [review]

**Pull request merged**